feature-detects/css/flexbox.js

```diff
@@ -6,7 +6,7 @@
   "tags": ["css"],
   "notes": [{
     "name": "The _new_ flexbox",
-    "href": "http://dev.w3.org/csswg/css3-flexbox"
+    "href": "https://drafts.csswg.org/css-flexbox-1/"
```

Comment:
@gregwhitworth while we are updating, would you consider the css drafts release to be the most correct one to link to in modernizr's docs?